memcached/SOURCES/memcached.service

```diff
@@ -4,9 +4,9 @@ Before=httpd.service
 After=network.target
 
 [Service]
-PIDFile=/var/run/memcached/memcached.pid
-ExecStart=/etc/init.d/memcached start
-ExecStop=/etc/init.d/memcached stop
+EnvironmentFile=-/etc/sysconfig/memcached
```

Comment:
Looks like path to sysconfig file is invalid

Reply:
It works (`-` means "ignore errors"), but for consistency of systemd configurations, I removed minus in front of the path.

Reply:
I didn't know that. Thanks for explanation.